gratipay/utils/fake_data.py

```diff
@@ -167,6 +167,22 @@ def fake_elsewhere(db, participant, platform):
                , extra_info=None
                 )
 
+def fake_payment(db, participant, team):
+    """Create fake payment
+    """
+    direction = ['to-team','to-participant']
+    if participant.username == team.owner:
+        direction = 'to-participant'
+    else:
+        direction = random.sample(['to-team','to-participant'],1)[0]
```

Comment:
Since we don't have takes yet - If the participant isn't an owner, the direction will always be `to-team`.